src/Npgsql.NetTopologySuite/Npgsql - Backup.NetTopologySuite.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Committed by mistake?

Reply:
VS is behaving badly on rebases. Will cleanup.